### PR TITLE
Add Hombox by hay-kot

### PR DESCRIPTION
**Pull request by centfo22**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/homebox-by-hay-kot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Homebox
+website_url: https://hay-kot.github.io/homebox/
+description: Homebox is the inventory and organization system built for the home user.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
```
Comment (KieranRobson) [inline software/homebox-by-hay-kot.yml:7]: > https://github.com/hay-kot/homebox/#quick-start
> https://hay-kot.github.io/homebox/quick-start/

```suggestion
platforms:
  - Docker
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Quick change to be made.

File: software/homebox-by-hay-kot.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Homebox
+website_url: https://hay-kot.github.io/homebox/
+description: Homebox is the inventory and organization system built for the home user.
```
Comment (nodiscc) [inline software/homebox-by-hay-kot.yml:3]:
```suggestion
description: Inventory and organization system built for the home user.
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/homebox-by-hay-kot.yml (hunk 2 above)
Comment (centfo22) [reply software/homebox-by-hay-kot.yml]: Added back Go to the platforms as it is written with Go and deployed with Docker.

Comment (KieranRobson) [reply software/homebox-by-hay-kot.yml]: >  Values for platform are the main server-side requirements for the software. Don't include frameworks or specific dialects.

Only Docker is needed

Comment (centfo22) [reply software/homebox-by-hay-kot.yml]: What is the downside in having more information? It is useful to know that it is written in Go. There are multiple other listed software that includes docker and the server side language it is written in.

Comment (KieranRobson) [reply software/homebox-by-hay-kot.yml]: Those services include deployment details for that language. 

The downside to more information is that if someone is looking to host this software, and they don't run docker, they can't use the documentation to host this software with Go.

If Go documentation is added, then Go should be added to platforms.

**Pull request closed without merging**